stdpopsim/catalog/HomSap/__init__.py

```diff
@@ -202,6 +202,37 @@
     )
     _species.add_genetic_map(_gm)
 
+###########################################################
+#
+# Annotations
+#
+#
+#
+###########################################################
+
+_an = stdpopsim.Annotation(
+    species=_species,
+    id="Ensembl_GRCh37_gff3",
+    description="Ensembl GFF3 annotations on GRCh37",
+    long_description="""
+        These are the complete GFF3 annotations from
+        Ensembl. Please see
+        ftp://ftp.ensembl.org/pub/grch37/current/gff3/homo_sapiens/README
+        """,
+    url=(
+        "http://ftp.ensembl.org/pub/grch37/current/gff3/"
+        "homo_sapiens/Homo_sapiens.GRCh37.87.gff3.gz"),
+    zarr_url=(
+        "https://stdpopsim.s3-us-west-2.amazonaws.com/"
+        "annotations/HomSap.zip"),
+    citations=[
+        stdpopsim.Citation(
+            year=2017,
+            author="Aken et al",
+            doi="http://dx.doi.org/10.1093/nar/gkw1104",
```

Comment:
Ensembl seem to have a publication every year. How will we choose which one to cite? (I guess you chose this one because it corresponds with release 87?)

Reply:
looks like ensembl wants us to cite their most recent paper, if we are using their most recent releases: 

https://uswest.ensembl.org/info/about/publications.html